### DISCUSSION TITLE
Drag up / down in lists

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#314] [Q&A] [answered]
Please activate again the drag and drop of waypoints again on the waypoint number at the left and again smooth by direct one tap. At the moment only at the hamburger menu is very bad. I've have to tap much too long until it recognizes that I want to move waypoints. This is not intuitive any more. Also why only at the left or right, I would want to move directly a waypoint when tapping on the text, so any horizontal position.

**Comment by devemux86** [comment]
Now we need to long press _anywhere_ on the row + drag up / down to reorder.

We can return to previous workflow: tap on left icon + drag up / down to reorder.

As we also have the swipe left / right gesture to delete the element,
we cannot drag up / down anywhere on the row, we can use again the left icon.

**Reply by DolomitesTom** [reply]
OK, why long press for drag n drop? Is it possible to just short tap, hold and drag? If not otherwise possible, then please additionally short tap & drag on the left.

**Reply by devemux86** [reply]
This was discussed with the participating users in the last Beta,
as the long press became free after moving the menu to the right icon.

But we can return the tap + drag up / down on the left icon, no problem. :slightly_smiling_face: 
(that's how it worked before)

**Reply by Ciutnik** [reply]
Mainly (or even only) because of the list of route waypoints, I suggested using the left icon to change the order by dragging. It's worth making this activity easier on this list, and therefore on everywhere.

**Reply by devemux86** [reply]
Agreed, I will change it.

But we should change it in all lists so users don't see different workflow in other lists.

**Comment by devemux86** [comment]
> we can return the tap + drag up / down on the left icon

Available again in:
- https://github.com/devemux86/cruiser/discussions/329

**Comment by devemux86** [comment]
Could we also (additionally) make it dragable when tapping on the text (complete frame)? Because then it could be used easier with the right thumb for right-handers.

_Originally posted by @DolomitesTom in https://github.com/devemux86/cruiser/discussions/319#discussioncomment-7146080._

**Reply by devemux86** [reply]
@DolomitesTom 

This could work with long press, as there is also the swipe left / right to delete items.

**Comment by devemux86** [comment, marked as answer]
> we can return the tap + drag up / down on the left icon

Available again in [Cruiser 3.0.34](https://play.google.com/store/apps/details?id=gr.talent.cruiser).
